src/framework/tooltip/_tooltip.scss

```diff
@@ -0,0 +1,50 @@
+
+$fieldMessage_arrowSide: 6px;
+$fieldMessage_bgColor: #fafdfe;
+$fieldMessage_borderColor: #dbe2e3;
+
+.tooltip {
+  padding: 15px 13px;
+}
+
+.tooltipParent {
+  display: inline-block;
+  position: relative;
+}
+
+.tooltipPosition {
+  display: none;
+  opacity: 0;
+  position: absolute;
+
+  &:after {
+    background-color: $fieldMessage_bgColor;
+    border: solid $fieldMessage_borderColor;
+    border-radius: 3px;
```

Comment:
I think `border-radius` is not needed here. With it, the corners of the arrow become rounded.